internal/services/redisenterprise/validate/redis_enterprise_cluster_sku_name.go

```diff
@@ -71,11 +71,14 @@ func RedisEnterpriseClusterSkuName(v interface{}, k string) (warnings []string,
 
 func getValidRedisEnterpriseClusterSkus() []string {
```

Comment:
FWIW this function can be replaced by `redisenterprise.PossibleValuesForSkuName()` which will give us all the possible values for the `SkuName` constant (which is the list below) - meaning we'll get these automatically in future SDK upgrades

Reply:
Thanks for your feedback. The code has been updated. Could you please take another look?